src/global.rs

```diff
@@ -19,15 +27,41 @@ use sync::list::List;
 lazy_static! {
     /// REGISTRIES is the head pointer of the list of mutator registries.
     pub static ref REGISTRIES: List<LocalEpoch> = List::new();
+    /// GARBAGES is a reference to the global queue of garbages.
+    pub static ref GARBAGES: Queue<(usize, Bag)> = Queue::new();
     /// EPOCH is a reference to the global epoch.
     pub static ref EPOCH: Epoch = Epoch::new();
 }
 
 
+#[inline]
+pub fn push_bag<'scope>(bag: &mut Bag, scope: &'scope Scope) {
```

Comment:
Maybe add a comment stating what this does? (pushes bag onto global queue and replaces the mutref'd argument with a new bag)